modules/cij/runner.py

```diff
@@ -340,6 +372,8 @@ def trun_to_file(trun, fpath=None):
 def trun_to_junitfile(trun, fpath=None):
     """Generate jUNIT XML from testrun YML"""
 
+    # pylint: disable=too-many-locals
+
```

Comment:
I guess the stamp code could be simplified and remove locals. E.g.
```
if trun.stamp:
  trun.stamp["begin"] = trun.stamp.get("begin", time.time())
  trun.stamp["end"] = trun.stamp.get("end", time.time())
```

Reply:
Ah yeah, that's a great simplification!

I think putting the variables into a dict instead of using real variables is kind of worsening the code just to satisfy pylint warnings. In my opinion the code would be better if try to eliminate a variable elsewhere in the scope or keep the disable hint :)
I'll try to :dagger: a variable elsewhere!

Reply:
I updated this and managed to simply even further. I think I'd personally keep the pylint-disable over what I came up with (because of readability), but I think both are okay

Reply:
But the stamp-field is already a dict, right? The proposal just does not use local-vars to give them a default-value?

Reply:
Ahhh, you're completely right. I don't know why I read this like we were introducing `trun.stamp["stamp_begin"]` and `trun.stamp["stamp_end"]`